lib/tasks/install_secrets.rake

```diff
@@ -84,3 +85,16 @@ def deep_populate(hash, keys, value)
     deep_populate(hash[keys[0]], keys[1..-1], value)
   end
 end
+
+def cast_to_number_if_number(string_or_array)
```

Comment:
do we have unit tests for this?

Reply:
Hmm, stole it from BIT/accounts, I'll look there

Reply:
There are now 4 diverging copies of install_secrets.rake, two of which contain this code, none of which have a spec. :/